xmlwriter/xmlwriter.php

```diff
@@ -519,7 +519,7 @@ public function flush ($empty = true) {}
  * @param string $uri <p>
  * The URI of the resource for the output.
  * </p>
- * @return bool|resource Object oriented style: Returns <b>TRUE</b> on success or <b>FALSE</b> on failure.
+ * @return false|resource|XMLWriter Object oriented style: Returns <b>TRUE</b> on success or <b>FALSE</b> on failure.
```

Comment:
https://github.com/php/php-src/blob/master/ext/xmlwriter/php_xmlwriter.c#L924 seems it can return true as well so reducing type from `bool` to `false` seems to be incorrect

Reply:
The ```if (self) {``` check will prevent this behavior for the procedural style. 

⇾ https://www.php.net/manual/en/function.xmlwriter-open-uri.php